perfkitbenchmarker/providers/openstack/os_disk.py

```diff
@@ -12,107 +12,154 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import json
 import logging
-import time
-
 
+from perfkitbenchmarker import disk
 from perfkitbenchmarker import errors
 from perfkitbenchmarker import flags
-from perfkitbenchmarker import disk
+from perfkitbenchmarker import vm_util
 from perfkitbenchmarker.providers.openstack import utils as os_utils
 
 FLAGS = flags.FLAGS
 
+REMOTE_BOOT_DISK_SIZE_GB = 50
+
+
+def CreateVolume(resource, name, image=None):
+  """Creates a remote (Cinder) block volume."""
+  vol_cmd = os_utils.OpenStackCLICommand(resource, 'volume', 'create', name)
+  vol_cmd.flags['availability-zone'] = resource.zone
+  if image:
+    vol_cmd.flags['image'] = image
+    vol_cmd.flags['size'] = GetImageMinDiskSize(resource, image)
+  if FLAGS.openstack_volume_size:
+    vol_cmd.flags['size'] = FLAGS.openstack_volume_size
+  else:
+    vol_cmd.flags['size'] = REMOTE_BOOT_DISK_SIZE_GB
```

Comment:
Maybe you should change it to use disk size spec from default_config_constrants.yaml file?